provider/gce/google/disk.go

```diff
@@ -201,30 +198,60 @@ type Disk struct {
 	// Id is an unique identifier google adds to the disk, it usually
 	// is not used in the API.
 	Id uint64
+
 	// Name is a unique identifier string for each disk.
 	Name string
-	// Description holds the description field for a disk, we store env UUID here.
+
+	// Description holds the description field for a disk, we used to
+	// store the model UUID here.
```

Comment:
Might be worth mentioning we still store other tags here since the don't conform to strict label rules?